tools/compare_v_performance_between_commits

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
What about making it at least POSIX compatible with the only dependency being `hyperfine`? I'll comment on all places where it's either not POSIX or unsafe :wink:.
```sh
#/bin/sh
```

Reply:
I would rather rewrite it in v, than support obscure shells and their syntax idiosyncrasies. 
Bash is already everywhere, and for a first prototype it works fine.

Reply:
Well, this won't work very well on some BSDs probably. Recently Solaris support was/is_being added and I can guarantee this won't run there. If not writing it in *V* right now, then I'd strongly recommend making this script safe in the first place (all the missing double quotes, etc.) and when at it, then just copy paste my code suggestions regarding POSIX compatibility (it's not that much code all in all, so I find it better to have it merged clean first rather than unclean :wink:).

Reply:
+1 for writing in V